Project.toml

```
@@ -4,7 +4,7 @@ authors = ["Eric Davies"]
 version = "3.0.0"
 
 [compat]
-julia = "0.7"
+julia = "~0.7, ~1"
```

```suggestion
julia = "1"
```
At this point we should drop 0.7, I think.

---

Travis needed updating too, so fixed in latest push.  I also removed `REQUIRE`.